README.md

```diff
@@ -539,8 +554,8 @@ BlogPost.get(123).then((data) => {
 });
 ```
 
-**options** parameter  
-The options object parameter has a **preserveOrder** property (default to false). Useful when an array of IDs is passed and you want to preserve the order of those ids in the results. 
```

Comment:
Here too. I think it would be best to revert this file and put back just what is relevant for validate :) It happened to me too when using an auto-fomatter on .md files. The 2 spaces are necessary for line breaks. thanks!